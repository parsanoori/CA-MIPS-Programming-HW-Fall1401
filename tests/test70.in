48
1005587307
617434430
330995248
275557669
139048200
145788958
73751027
472510530
396417942
465126937
510625798
164241504
844324181
1012886292
680672723
650370569
643312522
470868423
734107450
484847941
636890071
1031805941
458855522
653953995
984277132
155805275
614095455
229465262
914928408
666796
509468008
34318270
231153438
852034258
349213645
396117782
141760387
356893578
203479234
822761967
1004620451
57372068
71379270
683535847
335683069
318115804
729746865
686940888