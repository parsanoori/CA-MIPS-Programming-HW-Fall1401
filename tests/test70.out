666796
34318270
57372068
71379270
73751027
139048200
141760387
145788958
155805275
164241504
203479234
229465262
231153438
275557669
318115804
330995248
335683069
349213645
356893578
396117782
396417942
458855522
465126937
470868423
472510530
484847941
509468008
510625798
614095455
617434430
636890071
643312522
650370569
653953995
680672723
683535847
686940888
729746865
734107450
822761967
844324181
852034258
914928408
984277132
1004620451
1005587307
1012886292
1031805941